csharp/src/SeedLang/Interpreter/Opcode.cs

```diff
@@ -18,6 +18,7 @@ namespace SeedLang.Interpreter {
   // All the opcodes of SeedLang virtual machine.
   internal enum Opcode {
     MOVE,         // R(A) := RK(B)
+    LOADNONE,     // R(A), R(A+1), ..., R(A+B-1) := none
```

Comment:
1. I also think nil might be good for the low level naming. So I decide to name the opcode as `LOADNIL`, AST node as `NilConstantExpresison` and token type as `Nil`. But keep the name in SeedPyhton parser as `None`.
2. Lua uses inconsistent style in the VM implementation, like [A, B] or [A, A+B-2], for example, there is a mistake in: https://the-ravi-programming-language.readthedocs.io/en/latest/lua_bytecode_reference.html#op-loadnil-instruction
```
RETURN  A B return R(A), ... ,R(A+B-2)
// B=0 is used to expression to top of the stack
LOADNIL A B     R(A), R(A+1), ..., R(A+B) := nil
// Shall be R[B] instead of R[A+B]
// Sets a range of registers from R(A) to R(B) to nil.
// If a single register is to be assigned to, then R(A) = R(B).
```

I decide to use a consistent way in SeedVM that B or C is always the count of the registers.